### PR TITLE
fix regression of get_node_by_name change

### PR DESCRIPTION
**Pull request by pengwa**
initializer was not correctly retrieved with get_node_by_output. add related logic to do that.

### CODE REVIEW

File: tf2onnx/tfonnx.py [hunk 1]
```diff
@@ -2216,6 +2216,9 @@ def rewrite_incomplete_type_support(g, ops, impacted_ops):
             ignored_inputs = ignored_input_index.get(op.type)
             # insert casts on inputs if the runtime only supports float
             for i, input_node in enumerate(op.inputs):
+                #print(op.input[i])
```
Comment (guschmue) [inline tf2onnx/tfonnx.py]: I'm sure pylint is not going to like it :)

Comment (pengwa) [reply tf2onnx/tfonnx.py]: exactly. my bad.

Comment (pengwa) [reply tf2onnx/tfonnx.py]: BTW, seems pylint did not notice this. same with my local pylint result.

Comment (guschmue) [reply tf2onnx/tfonnx.py]: odd, I know it doesn't like #no_space ... oh well.

**Comment by pengwa** [comment]
thanks @guschmue

**Pull request merged**